### ISSUE TITLE
Update Assessment's Overview section

### ISSUE THREAD

**Issue by risanaka** [microsoft/accessibility-insights-web#287]
## Describe the bug
Users that read the overview section's summary paragraph don't always understand how assessment works.

**To Reproduce**

1. Open Accessibility Insights for Web.
1. Go to Assessment.
1. Go to Overview

## Expected behavior
This summary indicates the overall accessibility of the website or web app, based on a combination of automated and manual tests that cover all the success criteria described by the Web Content Accessibility Guidelines (WCAG) 2.0 for Level AA conformance. Each manual test covers a set of related accessibility requirements, with provided step-by-step testing instructions and guidance. Where possible, Accessibility Insights for Web assists the manual testing process by generating a list of potential issues to evaluate and highlighting them on the screen. Accessibility Insights for Web also allows you to manually record instances of failure. 

## Extension (please complete the following information)

- OS: [e.g. Windows/Mac]
- Version [e.g. 1.140.1]
- Chrome Version [e.g 71.0]
- Target Page [e.g www.bing.com]

## Are you willing to submit a PR?

If asked, will you be willing to submit a PR to fix this bug? Yes/No

## Did you search for similar existing issues?

Did you search for similar issues in our github issues or on [stackoverflow](https://stackoverflow.com/questions/tagged/accessibility-insights)
